src/media/js/elements/nav.js

```diff
@@ -0,0 +1,182 @@
+define('elements/nav',
+    ['core/log', 'core/z'],
+    function(log, z) {
+
+    var logger = log('elements/nav');
+    console.log('initializing');
+
+    var MktNavElement = document.registerElement('mkt-nav', {
+        prototype: Object.create(HTMLElement.prototype, {
+            // Toggle the class specified by this.statusVisibleClass on
+            // this.statusElement. If an optional argument is passed, it will
+            // force the class to be added or removed based on the truthiness of
+            // that value.
+            // 
+            // Return `this`, for chaining.
+            toggle: {
+                value: function(bool) {
+                    if (typeof bool !== 'undefined') {
+                        this.statusElement.classList.toggle(this.statusVisibleClass, bool);
+                    } else {
+                        this.statusElement.classList.toggle(this.statusVisibleClass);
+                    }
+
+                    return this;
+                },
+            },
+
+            // Toggle the class specified by this.statusSubnavVisibleClass on
+            // this.statusElement. If an optional argument is passed, it will
+            // force the class to be added or removed based on the truthiness of
+            // that value.
+            // 
+            // Return `this`, for chaining.
+            toggleSubnav: {
+                value: function(bool) {
+                    if (typeof bool !== 'undefined') {
+                        this.statusElement.classList.toggle(this.statusSubnavVisibleClass, bool);
+                    } else {
+                        this.statusElement.classList.toggle(this.statusSubnavVisibleClass);
+                    }
+                    return this;
+                },
+            },
+
+            // Return the class to be added to this.statusElement when the child
+            // <mkt-nav-root> should be visible.
+            statusVisibleClass: {
+                get: function(){
```

Comment:
Right, to just set the value to a string instead of a getter that always returns the same string.

```js
statusVisibleClass: {value: 'mkt-nav--visible'},